### ISSUE TITLE
mainboard mount MKS SGEN_L

### ISSUE THREAD

**Issue by doug-scott** [bgdog1/3dPrinterCase#7]
I see in the readme.md that these two mounts were created a few months ago:

6/2/2020
Added Mainboard Mount V4_0 - MKSGenL_centered
Added Mainboard Mount V4_0 - MKSGenL_offset

I cannot find them.  Are they still applicable to v5 cases?

**Comment by doug-scott** [comment]
Have the rest of the case printed, just need the mainboard mounts for my MKS SGEN_L board.  The readme.md says they were added back in June, but I cannot locate them on Thingiverse, Facebook or here.

**Comment by doug-scott** [comment]
I found the two mainboard mounts by stumbling onto 'bgdog1' changes and going back to June 2, finding where they were inserted into v4 components.  I do not know if they are v4 specific yet, haven't printed them yet.  Don't see why not, they are just simple flat adaptor plate.  Also grabbed the v4 front and rear standard body shells just in case.  Those cases actually fit my system better, they don't have any of the side openings, and since the mainboard mount does not insert through the side like the other mounts do, that means I don't need them.  The front display mount suits my needs, as does the lower grille when the SD card mounts are added to the slotted grill from within S3D.  I do not know how to use any of the graphic apps, so I just inserted 2 of the SD card adaptor mounts into the slotted grill, then print that part out, and trim the excess slotted grill that is printed into the SD mounts.  I have a SD mount on either side of the grill to allow access to the SD card sockets on the TFT and the mainboard.
